build.sbt

```diff
@@ -237,11 +236,11 @@ lazy val storehausLevelDB = module("leveldb").settings(
   fork in Test := true
 ).dependsOn(storehausCore % "test->test;compile->compile")
 
-lazy val storehausKafka08 = module("kafka-08").settings(
```

Comment:
The main goal was to move to the new producer which is supported in 0.8 and 0.9 and having only one jar which would be named `storehaus-kafka` instead of having one per version.
What do you think?